app/models/generic_event/incident.rb

```diff
@@ -3,14 +3,21 @@ class Incident < GenericEvent
     LOCATION_ATTRIBUTE_KEY = :location_id
 
     def self.inherited(child)
-      child.details_attributes :supplier_personnel_numbers, :vehicle_reg, :reported_at
+      child.details_attributes :supplier_personnel_numbers, :vehicle_reg, :reported_at, :fault_classification
```

Comment:
if this is needed on every incident request, should we add a required validation, similar to `SupplierPersonnelNumberValidations`?

Reply:
We should, yeah. I've actually documented that it needs to be present so :)